osu.Game.Rulesets.Osu/Skinning/Legacy/LegacyMainCirclePiece.cs

```diff
@@ -22,9 +23,10 @@ public class LegacyMainCirclePiece : CompositeDrawable
     {
         public override bool RemoveCompletedTransforms => false;
 
-        private readonly string priorityLookup;
         private readonly bool hasNumber;
 
+        private string priorityLookup;
```

Comment:
Might be good to rename this to `priorityLookupPrefix` and potentially add some xmldoc explaining it briefly. It's a bit weird as a variable name.